test/css-validator_test.js

```diff
@@ -19,6 +19,12 @@ function runValidateCss() {
   });
 }
 
+function expectValid(data) {
+  expect(data.validity).to.equal(true);
+  expect(data.errors).to.deep.equal([]);
+  expect(data.warnings).to.deep.equal([]);
+}
```

Comment:
This is a personal preference but I hate consolidated assertions. It loses clarity of what's being asserted inside of the test case which can introduce bugs. Can we please use the long form in both locations?